### ISSUE TITLE
the example project does not compile

### ISSUE THREAD

**Issue by murad1981** [shyambhat/InstagramKit#236]
after doing pod install for the library and the example project, there is a build error when trying to compile it with XCode 7 (the error is something like: -ipods not found ...)

please check out the project for errors.

**Comment by gugges** [comment]
You probably just need to switch the target to InstagramKit-Example. You can also type "pod try InstagramKit" from terminal.

**Comment by murad1981** [comment]
already done so but I keep getting this error: 
![screen shot 2016-06-20 at 9 03 08 am](https://cloud.githubusercontent.com/assets/1199045/16184555/e74b8158-36c5-11e6-8d8d-c1dcb3db44f9.png)

**Comment by gugges** [comment]
You need to run pod install in the /InstagramKit/InstagramKit-Example directory and then open the xcworkspace, not the xcproject in that same location.

**Comment by murad1981** [comment]
I've already done this exactly

**Comment by gugges** [comment]
Pulled down the latest master and can confirm this works fine. Are you opening the workspace in the /InstagramKit/InstagramKit-Example directory or just the /InstagramKit/ directory? Also, please confirm if my suggestion to run "pod try Instagram-Kit" from terminal works?

**Comment by kvnbautista** [comment]
@murad1981 or you can just open a new project, stuff all the files from the repository even the pod files folder. Then Compile it. Whichever is easier for you.

**Comment by murad1981** [comment]
i've tried pod try Instagram-Kit in InstagramKit-master and InstagramKit-Example directories but terminal gave me the same error for both cases: [!] Unable to find a specification for `Instagram-Kit`

**Comment by gugges** [comment]
pod try works from any directory in bash if cocoapods is installed. Your issue is that it should be pod try InstagramKit (no dash) not pod try Instagram-Kit

**Comment by MuezzaGit** [comment]
Greetings, I am facing the same error that Murad1981 is reporting. Have tried the steps recommend but still does not compile. Any suggestions appreciated. Thanks.

**Comment by qqc** [comment]
Podfile definition changed in newer versions of Cocoapods, so if pod install results in the following error:

```
$ pod install
Analyzing dependencies
[!] The dependency `InstagramKit (from `../`)` is not used in any concrete target.
The dependency `InstagramKit/UICKeyChainStore (from `../`)` is not used in any concrete target.
```

Try the following podfile instead:

```
platform :ios, '9.0'

target 'InstagramKit-Example' do
  use_frameworks!

  # Pods for InstagramKit-Example
  pod "InstagramKit", :path => '../'
  pod "InstagramKit/UICKeyChainStore", :path => '../'

end
```

@murad1981 To address the `Library not found for -lPods` issue, remove libPods.a from Linked Frameworks and Libraries under General in the project settings.

<img width="995" alt="screen shot 2016-06-22 at 20 13 38" src="https://cloud.githubusercontent.com/assets/3880807/16280108/f5c795cc-38b5-11e6-8c37-306092804653.png">

**Comment by ricsantos** [comment]
I have made PR #273 which resolves this.

**Comment by shyambhat** [comment]
Fixed with InstagramKit 4.0. Thanks all.
